### ISSUE TITLE
Arduino version?

### ISSUE THREAD

**Issue by beckmx** [justintconroy/MdbBillValidator#5]
Hello what version of arduino does it run on?, can I just place the content in the library section?

**Comment by justintconroy** [comment]
I ran this on an Arduino Uno. This is a library, so you would have to place it in the libraries folder of your arduino IDE installation (assuming you're using the Arduino IDE, that is). It's been a while since I've touched this project, so I may not be of much more help than that.

**Comment by beckmx** [comment]
yeah no problem, so basically the mdb port is generic and then you use the validation on top of that right? I guess you can spy on the bus if you connect it directly?

**Comment by justintconroy** [comment]
I'm not sure I understand what you mean by validation in this particular case. If I recall correctly, I think I did implement quite a few commands that were specific to a bill validator. I wasn't trying to make a thing that would work with any MDB device. I only ever had a bill validator.

**Comment by beckmx** [comment]
I see, so you created your own driver to understand the bill validator, simulating a vending I suppose

**Comment by justintconroy** [comment]
I created it because I actually wanted to build a machine that would accept cash and update a system with payment information. I just never actually finished that part of the project. ;-)

**Comment by Tarcontar** [comment]
hi, so have you actually tested your code with a real bill validator? (since i cant really get it to work with mine)^^

**Comment by beckmx** [comment]
I tested with a real bill validator and it didn't work :(

2017-01-30 17:00 GMT-06:00 Tarcontar <notifications@github.com>:

**Comment by Tarcontar** [comment]
any idea what might be the problem? the 9bit serial?

**Comment by beckmx** [comment]
nfi to be honest

**Comment by justintconroy** [comment]
It's been a couple of years since I toyed with this, but I did have an actual bill validator to experiment with when I was first developing this code. If you open a new issue describing the problems you're having, I might be able to help you. You should describe all the hardware you're using in the issue you create.
